### PR TITLE
Day 1 Release

### PR DESCRIPTION
**Pull request by reactivepixel**

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -45,10 +45,16 @@ module.exports = function() {
...
   }
 
+  function _scrubData(dirtyData){
+    var cleanData = dirtyData;
+    return cleanData;
+  }
+
   return {
     hayTest: _hayTest,
     debug: _debug,
```
Comment (reactivepixel) [inline lib/util.js:60]: You named that poorly.

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+var gulp = require('gulp'),
+nodemon = require('gulp-nodemon');
+var shell = require('gulp-shell');
+
+//Task to initiate JSdocs
+gulp.task('js-doc', shell.task(['jsdoc src/server.js']));
+
+// Task to move JSdoc 'out' folder to correct place
+gulp.task('default', ['dev','js-doc'], function() {
+  console.log("Moving all files for jsdoc ");
```
Comment (reactivepixel) [inline gulpfile.js]: Console logs are not good, use util.debug(title,obj);

File: src/routes/index.js [hunk 3]
```diff
@@ -1,6 +1,20 @@
 module.exports = function(express) {
+  var express = require('express');
```
Comment (reactivepixel) [inline src/routes/index.js]: Express is already established in the line before this, so this line is redundant. Please remove.

File: src/routes/index.js [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
+  var histories = require('../models/histories.js');
```
Comment (reactivepixel) [inline src/routes/index.js]: Note that all other models are referred to in the singular. Please adjust.

File: src/routes/index.js [hunk 5]
```diff
@@ -1,6 +1,20 @@
...
+  var db = require('../models/db.js');
   var router = express.Router();
 
+
+// NOTES___________________________________________________
```
Comment (reactivepixel) [inline src/routes/index.js]: This does not match the format for commenting as defined in the airbnb style guide. Also, distill your comments down to a shorter and more cohesive statement. Please refine.

File: src/models/histories.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  var db = require('./db.js');
+  var sequelize = db.connection;
+  var util = require('../../lib/util');
+
+// Create
+  function _create(payload, err, success) {
+    var cleanData = util.scrubData(payload);
+    db.histories.create(cleanData).then(success).catch(err);
+  }
+// Update
```
Comment (reactivepixel) [inline src/models/histories.js]: Does not match Comment Formatting in the airbnb style guide. Also, all comments on this file are inconsistent.

File: src/models/db.js [hunk 7]
```diff
@@ -33,6 +51,45 @@ module.exports = function() {
     },
     phone: {
       type: Sequelize.STRING
+    },
+
+    //added ability for user to set gender in DB on registration to generate recommendations based on gender
```
Comment (reactivepixel) [inline src/models/db.js:60]: The comments on this file, while robust, do not need to explain the context in which the code was added. Comments need only be applied for two major reasons. 
1. If the code is sufficiently complex and needs to be explained
2. If the documentation engine (jsdocs) requires it

**Comment by reactivepixel** [comment]
@jc-roll if something should be a devDependency or normal dependency is governed only by our use of it. The question to ask: Is the package going to be used by our dev environment to do something for us (like gulp and jsdocs) or is it going to be used in production, something that's critical to the system (like express or sequelize).

I am just staring to look at ger, so I'm not certain just yet. Also @seanedw1 's team should have documented how to use ger, as far as I know, it just showed up and absolutely polluting the console 😢

File: src/models/db.js (hunk 7 above)
Comment (reactivepixel) [inline src/models/db.js:60]: This relates to issue #7

File: src/models/db.js [hunk 8]
```diff
@@ -102,6 +181,25 @@ module.exports = function() {
...
   // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  //    Review
+  // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  // This is making the table in SQL with 3 fields for now: name, post, & star rating.
+  var _review = _sequelize.define('review', {
+    name: {
+      type: Sequelize.STRING
+    },
+    post: {
```
Comment (reactivepixel) [inline src/models/db.js:192]: Change post to `content` or some other better descriptor.

File: src/routes/reviews.js [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+/* =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+  SAVE EDITED REVIEWS
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=- */
+/* Save any edits made to your review
+Required for UPDATE part of CRUD */
+
+       // console.log(query.sql); Test to get raw query
+
+    });
+};
```
Comment (Alucidisland) [inline src/routes/reviews.js]: Please fix this code if you don't see whats wrong, hit me up on slack @lindsayhampton

**Comment by reactivepixel** [comment]
Abandoning Request

**Pull request closed without merging**